notify/impl.go

```diff
@@ -584,7 +584,7 @@ func (n *Hipchat) Notify(ctx context.Context, as ...*types.Alert) error {
 	}
 
 	req := &hipchatReq{
-		From:          tmplText(n.conf.From),
+		From:          truncate(tmplText(n.conf.From), 64),
```

Comment:
From just reading the code it's not completely clear why 64 chars. If I git blame I would find out. Is that the preferred way? Maybe either add a comment or move the 64 to a constant to describe it. Wdyt?